libkbfs/block_retrieval_queue.go

```diff
@@ -213,6 +215,8 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 func (brq *blockRetrievalQueue) CacheAndPrefetch(ctx context.Context,
 	ptr BlockPointer, block Block, kmd KeyMetadata, priority int,
 	lifetime BlockCacheLifetime, hasPrefetched bool) (err error) {
+	didUpdateCh := make(chan struct{})
+	donePrefetch := hasPrefetched
```

Comment:
Please update these `donePrefetched` variables as well, for consistency.  Although maybe for now we should always be setting it to `false` anyway?

Can we also change `hasPrefetched` to `triggeredPrefetch` like in the interface?